tests/scenario/test_ingressed_tracing.py

```diff
@@ -0,0 +1,30 @@
+import json
+import socket
+
+import pytest
+from charms.tempo_k8s.v1.charm_tracing import charm_tracing_disabled
+from scenario import Container, Relation, State
+
+
+@pytest.fixture
+def base_state():
+    return State(leader=True, containers=[Container("tempo", can_connect=False)])
+
+
+def test_external_url_present(context, base_state):
+    # WHEN ingress is related with external_host
+    tracing = Relation("tracing", remote_app_data={"receivers": "[]"})
+    ingress = Relation("ingress", remote_app_data={"external_host": "1.2.3.4", "scheme": "http"})
+    state = base_state.replace(relations=[tracing, ingress])
+
+    with charm_tracing_disabled():
+        with context.manager(getattr(tracing, "created_event"), state) as mgr:
```

Comment:
if you're not planning to use mgr.charm, you should:

```suggestion
        context.run(getattr(tracing, "created_event"), state)
        ```